src/main/java/io/antmedia/rest/BroadcastRestService.java

```diff
@@ -594,8 +606,115 @@ public Result validateTokenV2(@ApiParam(value = "Token to be validated", require
 		return new Result(result);
 	}
 
+	@ApiOperation(value = "Add jwt token to blacklist. If added, success field is true, "
+			+ "if not added success field false", response = Result.class)
+	@POST
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/jwt-black-list")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Result blackListOrWhitelistJwt(@ApiParam(value = "jwt to be added to blacklist.", required = true) @QueryParam("jwt") String jwt,
+										  @ApiParam(value = "if true whitelist jwt (unblacklist)", required = false) @QueryParam("whiteList") boolean whiteList)
```

Comment:
Sending a POST to `/jwt-black-list?whiteList=true` seems a bit weird to me, just read back the docs: "unblacklist".. Would it not be clearer if one had to send a DELETE request to the collection `/jwt-black-list` to get something removed?

Reply:
this is not removing anything from disk thats why i made it a post call.
But we can also make it a delete

Reply:
It is not the disk that matters, but it does remove someone from the blacklist, so I think a DELETE would be a better choice.